apps/web/next.config.js

```diff
@@ -77,7 +77,7 @@ const contentSecurityPolicy = {
     ccaLiteDomains,
     walletconnectDomains,
   ],
-  'worker-src': ['blob:'],
```

Comment:
what was using blob?

Reply:
There was an error with Nextjs trying to load a worker via blob:, it seems the solution is to use self: instead